lib/shoppingcart.rb

```diff
@@ -41,20 +38,19 @@ def products_by_category(category)
      end
   end
 
-
   def percentage_occupied
-   (((total_number_of_products.to_f / capacity))*100).round(2)
+   ((total_number_of_products.to_f / capacity.to_f) * 100).round(2)
```

Comment:
If you make both of these numbers a float, then you can avoid the issues that arise when dividing an integer and a float.